packages/rocketchat-notifications/server/lib/Notifications.js

```diff
@@ -74,7 +75,7 @@ class Notifications {
 			// });
 			if (Subscriptions.findOneByRoomIdAndUserId(roomId, this.userId) != null) {
 				const subscriptions = Subscriptions.findByRoomIdAndNotUserId(roomId, this.userId).fetch();
-				subscriptions.forEach((subscription) => this.notifyUser(subscription.u._id, e, ...args));
+				subscriptions.forEach((subscription) => self.notifyUser(subscription.u._id, e, ...args));
 			}
 			return false;
```

Comment:
Shouldn't this return true on success?

I'm getting a nosub message when sending stream-notify-room-users on `"params":["myid/webrtc",{"args":[],"useCollection":false}]`